kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1199,4 +1199,13 @@ object KyuubiConf {
       .version("1.5.0")
       .stringConf
       .createOptional
+
+  val ENGINE_FLINK_MAX_ROWS: ConfigEntry[Int] =
+    buildConf("engine.flink.max.rows")
+      .doc("Max rows of Flink query results. For batch queries, rows that exceeds the limit " +
+        "would be ignored. For streaming queries, the query would be canceled if the limit " +
+        "is reached.")
+      .version("1.5.0")
+      .intConf
+      .createWithDefault(500)
```

Comment:
IMO, this could refer to the default value of the `SqlClientOptions#EXECUTION_MAX_TABLE_RESULT_ROWS` option.